rocks-db/src/bin/migrate_slots_db/main.rs

```diff
@@ -0,0 +1,143 @@
+    let mut batch = rocksdb::WriteBatchWithTransaction::<false>::default();
+    for (k, v) in values.drain(..) {
+        let serialized_value = C::encode(&v).map_err(|e| StorageError::Common(e.to_string()))?;
+        batch.put_cf(
+            &backend.cf_handle(C::NAME).unwrap(),
+            C::encode_key(k.clone()),
+            serialized_value,
+        )
+    }
+    backend.write(batch)?;
+    Ok(())
+}
+
+#[tokio::main(flavor = "multi_thread")]
+async fn main() {
+    let args = Args::parse();
+    let tasks = Arc::new(Mutex::new(JoinSet::<Result<(), JoinError>>::new()));
+    let red_metrics = Arc::new(RequestErrorDurationMetrics::new());
+    let cf_descriptors = Storage::cfs_to_column_families(
+        SlotStorage::cf_names()
+            .into_iter()
+            .chain(std::iter::once(RawBlockDeprecated::NAME))
+            .collect(),
+    );
+    let db = Arc::new(
+        DB::open_cf_descriptors(
+            &Storage::get_db_options(),
+            args.slots_db_path.clone(),
+            cf_descriptors,
+        )
+        .expect("open rocks slot storage to migrate"),
+    );
+    let slot_storage = SlotStorage::new(db, tasks, red_metrics);
+    eprintln!("Opened slots database in primary mode at {}", args.slots_db_path);
+    let mut iter = slot_storage.db.raw_iterator_cf(
+        &slot_storage.db.cf_handle(RawBlockDeprecated::NAME).expect("get raw blocks cf handle"),
+    );
+    let batch: Arc<Mutex<Vec<(u64, RawBlock)>>> =
+        Arc::new(Mutex::new(Vec::with_capacity(WRITE_BATCH_SIZE)));
+    let (tx, mut rx) = tokio::sync::mpsc::channel::<(u64, RawBlock)>(WRITE_BATCH_SIZE);
+    if let Some(start_slot) = args.start_slot {
+        iter.seek(RawBlockDeprecated::encode_key(start_slot as u64));
+    } else {
+        iter.seek_to_first();
+    }
+    let start = std::time::Instant::now();
+    let worker_semaphore = Arc::new(Semaphore::new(PARALLEL_WORKERS as usize));
+    let mut js = tokio::task::JoinSet::<()>::new();
+    let handle = tokio::task::spawn({
+        let batch = batch.clone();
+        let slot_storage = slot_storage.clone();
+        async move {
+            while let Some((slot, raw_block)) = rx.recv().await {
+                let mut batch = batch.lock().await;
+                batch.push((slot, raw_block));
+                if batch.len() >= WRITE_BATCH_SIZE {
+                    let ((start_slot, _), (end_slot, _)) =
+                        (batch.first().unwrap(), batch.last().unwrap());
+                    eprintln!("Writing slots {} through {} to rocksdb...", start_slot, end_slot);
+                    let db = slot_storage.db.clone();
+                    put_batch_vec::<RawBlock>(db, &mut *batch)
+                        .expect("raw blocks batch to be inserted into rocksdb");
+                }
+            }
+        }
+    });
+
+    let mut keys_processed = 0u32;
+
+    while iter.valid() {
+        let (k, v) = iter.item().unwrap();
+        let (k, v) = (k.to_vec(), v.to_vec());
```

Comment:
```suggestion
let (k, v) = (iter.key().unwrap().to_vec(), iter.value().unwrap().to_vec());
```

Reply:
IMO too minor of a change to boil the ocean with CI for 15 minutes